client/app/scripts/services/pusher.js

```diff
@@ -8,10 +8,18 @@
  * Service in the clientApp.
  */
 angular.module('clientApp')
-  .service('channelManager', function ($q, $http, $timeout, $pusher, $log, Config, Auth) {
+  .service('channelManager', function ($q, $http, $timeout, $pusher, $log, Config, $rootScope, Auth) {
 
     var channels = {};
 
+    // A private cache key.
+    var cache = {};
+
+    this.pusher = null;
+
+    // Update event broadcast name.
+    var broadcastUpdateEventName = 'ShoovNewPusherObject';
```

Comment:
we're not using it - and we don't need to. Clear cache shouldn't create a new connection.